### PR TITLE
Actual location data

### PR DESCRIPTION
**Pull request by danielpowell4**
- Adds starting location index page (prev called signup)
- Restyles cards
- Ensures feeds pulled from dashboard
- Adds general-purpose 'classes' page

### CODE REVIEW

File: src/components/CampDetails.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import StyledChecks from "./shared/StyledChecks";
+import BoxWithLogo from "./shared/BoxWithLogo";
+
+const activities = [
+  {
+    title: "Project-Based Coding",
+    content:
```
Comment (aasalazar23) [inline src/components/CampDetails.js:8]: I didn't add this because we weren't going to use it anymore

Comment (danielpowell4) [reply src/components/CampDetails.js:8]: I'm in a mindset of trying to port the existing signup site over as close to 1:1 as possible

File: src/templates/signup-page.js [hunk 2]
```diff
@@ -210,10 +131,19 @@ export const SignupPageTemplate = ({
...
     frontmatter: { title, seoDescription, pageBuilder },
   } = data.markdownRemark;
-  const { edges } = data.allMarkdownRemark;
+  const experienceLevels = data.experienceLevelQuery.experienceLevels?.map(
+    (levelNode) => {
+      return {
+        ...levelNode.frontmatter,
+        slug: levelNode.fields.slug,
+      };
+    }
```
[the remark this answers is not in the document]
Comment (danielpowell4) [reply src/templates/signup-page.js]: In my current opinion, data flow and variable naming are one of the most important parts of code

Moving beyond getting it to work and striving to do it nice is a level up

**Comment by danielpowell4** [comment]
It works!

**Pull request merged**